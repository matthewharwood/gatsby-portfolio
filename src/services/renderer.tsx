


export const pageRenderer = () => {
  return true;
}