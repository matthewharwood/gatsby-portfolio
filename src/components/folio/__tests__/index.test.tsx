import React from 'react';
import renderer from 'react-test-renderer';
import { Folio } from '../';

describe('Should create Nav with children', () => {
  it('renders correctly', () => {
    const FIXTURE = {
      text: 'Hello',
      index: '01',
      to: '/hello',
    };
    const tree = renderer
      .create(
        <Folio
          headline={'Good Morning Harwood'}
          subheadline={'A creative playground by'}
          link={{ href: '#', text: 'Matthew Harwood' }}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
