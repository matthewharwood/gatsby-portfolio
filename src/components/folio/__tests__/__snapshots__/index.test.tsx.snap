// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should create Nav with children renders correctly 1`] = `
<footer
  className=""
>
  // @ts-ignore
  <span
    className=""
    data-baseweb="typo-label4"
  >
    Good Morning Harwood
  </span>
  <p
    className=""
    data-baseweb="typo-p4"
  >
    <span
      className=""
      data-baseweb="typo-label4"
    >
      A creative playground by
    </span>
    // @ts-ignore
    <a
      className=""
      data-baseweb="typo-label4"
      href="#"
    >
       
      Matthew Harwood
    </a>
  </p>
</footer>
`;
